### PR TITLE
Borealis

### PR DESCRIPTION
**Pull request by thisac**

### CODE REVIEW

**Comment by isaacdevlugt** [comment]
Great job guys! 🥳

File: doc/introduction/photonic_hardware.rst [hunk 1]
```diff
@@ -111,6 +111,18 @@ Tutorials
 For more details on submitting jobs to photonic hardware, check out the following
 tutorials.
 
+.. gallery-item::
+    :description: :doc:`demos/tutorial_borealis_quickstart`
```
Comment (josh146) [inline doc/introduction/photonic_hardware.rst:115]:
```suggestion
    :description: `Borealis quickstart <https://strawberryfields.ai/photonics/demos/tutorial_borealis_quickstart.html>`__
```

**Review by fab1an-q — APPROVED** [review]
Great work everyone! Is this the biggest PR in Strawberry Fields history?

File: doc/introduction/photonic_hardware.rst [hunk 2]
```diff
@@ -111,6 +111,18 @@ Tutorials
 For more details on submitting jobs to photonic hardware, check out the following
 tutorials.
 
+.. gallery-item::
+    :description: `Borealis quickstart <https://strawberryfields.ai/photonics/demos/tutorial_borealis_quickstart.html>`__
+    :figure: _static/borealis_3d.png
+
+.. gallery-item::
+    :description: :doc:`demos/tutorial_borealis_beginner`
```
Comment (josh146) [inline doc/introduction/photonic_hardware.rst:119]:
```suggestion
    :description: `Operating --- Borealis <demos/tutorial_borealis_beginner>`__
```

File: doc/introduction/photonic_hardware.rst [hunk 3]
```diff
@@ -111,6 +111,18 @@ Tutorials
...
+    :figure: _static/borealis_schematic_simple.png
+
+.. gallery-item::
+    :description: :doc:`demos/tutorial_borealis_beginner`
```
Comment (josh146) [inline doc/introduction/photonic_hardware.rst:123]:
```suggestion
    :description: `Operating Borealis --- Advanced <https://strawberryfields.ai/photonics/demos/tutorial_borealis_beginner.html>`__
```

**Pull request merged**